src/ocamlorg_frontend/pages/packages_autocomplete_fragment.eml

```diff
@@ -1,31 +1,84 @@
+(*
+  Package autocomplete works like this:
+  1. user types search terms into the search input
+  2. htmx fetches the fragment below (see function [render])
+     and inserts it into the HTMLElement selected by [target_sel].
+     While htmx's HTTP request is in progress, the HTMLElement
+     with selected by [indicator_sel] is shown.
+  3. user uses arrow keys to navigate the top 5 search results,
+     which present quick access to both the package overview page
+     and the documentation.
+
+  Note:
+  - when there are at most five search results, pressing the
+    Enter key in the input field sends you to the overview
+    page of the first search results
+ *)
```

Comment:
Are there any other places where this could be stored? Some doc, the commit message? Not sure about this

Reply:
I kept the one part that you need to know when you read the code as a comment and everything else is a commit message.